src/components/App/App.js

```diff
@@ -1,11 +1,15 @@
 import { useState } from 'react';
+import { Routes, Route } from 'react-router-dom';
 import NavBar from '../NavBar/NavBar';
 import LandingPage from '../LandingPage/LandingPage';
 import Map from '../Map/Map';
 import RecyclingTips from '../RecyclingTips/RecyclingTips';
 import SearchBar from '../SearchBar/SearchBar';
 import Footer from '../Footer/Footer';
 import './App.css';
+import Creators from '../Creators/Creators'
```

Comment:
The. creators page looks really nice and adds a nice extra to our project.  I like the idea of using a route here so that the user only sees our profiles if they want to.